### PR TITLE
New Settings UX

### PR DESCRIPTION
**Pull request by niels9001**
This PR introduces a new UX for the Settings page that is inline with Windows 11, inbox applications and the Windows Design Guidelines.

It uses the Windows Community Toolkit SettingsCard and SettingsExpander.. more information here: https://github.com/CommunityToolkit/Labs-Windows/discussions/129

## Description

GalleryUX

### CODE REVIEW

File: WinUIGallery/standalone.props [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <Project>
   <PropertyGroup>
     <!-- The NuGet versions of dependencies to build against. -->
-    <WindowsAppSdkPackageVersion>1.2.221109.1</WindowsAppSdkPackageVersion>
-    <SamplesTargetFrameworkMoniker>net6.0-windows10.0.18362.0</SamplesTargetFrameworkMoniker>
+    <WindowsAppSdkPackageVersion>1.2.221209.1</WindowsAppSdkPackageVersion>
+    <SamplesTargetFrameworkMoniker>net6.0-windows10.0.19041.0</SamplesTargetFrameworkMoniker>
```
Comment (chingucoding) [inline WinUIGallery/standalone.props:5]: Why is this needed?

**Review by chingucoding — COMMENTED** [review]
I think you want to merge main into this again since this PR doesn't currently compile. Also not sure if having a dependency for settingscard and settingsexpander is worth it. What are your thoughts @gregwoo-microsoft and @bpulliam ?

**Review by gregwoo-microsoft — APPROVED** [review]
Thanks for implementing this PR @niels9001! This brings some much-needed consistency when compared to our other inbox apps + what is mentioned in the Windows Design guidelines.
